examples/demo.js

```diff
@@ -1,39 +1,48 @@
 import React, { Component } from 'react'
 import clsx from 'clsx'
 import Villain from '@/index.js'
-import {version} from '../package.json';
+import { version } from '../package.json'
 import './demo.css'
+import ToggleThemeButton from './ToggleThemeButton'
 
 class Demo extends Component {
-
   state = {
     file: '/build/testFile/Example-archive.zip',
     options: {
       preview: 0,
-      theme: {type: 'select', value: 'Light', options: ['Dark', 'Light']},
+      theme: { type: 'select', value: 'Light', options: ['Dark', 'Light'] },
       mangaMode: false,
       allowFullScreen: true,
       autoHideControls: false,
       allowGlobalShortcuts: false,
-    }
+    },
+    theme: 'light',
   }
 
   constructor(props) {
     super(props)
   }
 
+  toggleTheme = () => {
+    const theme = this.state.theme === 'light' ? 'dark' : 'light'
+    document.documentElement.classList.add('color-theme-in-transition')
+    this.setState({ theme })
+    document.documentElement.setAttribute('data-theme', theme)
+    window.setTimeout(() => {
+      document.documentElement.classList.remove('color-theme-in-transition')
```

Comment:
Not sure if this class is necessary but, is better to use a flag to toggle this:

```jsx
<Component className={ flag ? 'class' : 'otherClass'  } />
```

Or better yet use the `clsx` module